pnpm-workspace.yaml

```diff
@@ -2,4 +2,6 @@ packages:
   - "docs"
   - "xsuite"
   - "xsuite-simulnet"
+  - "xsuite-chainsimulator"
   - "contracts/*"
+  - "xsuite-chain-simulator-temp"
```

Comment:
Will remove now to no longer have this PR in draft.

I made a backup of this here:
https://github.com/buidly/xSuite/tree/chain-simulator-temp/xsuite-chain-simulator-temp

Reply:
While the PR is not merged, it would be good to keep them in the branch to ensure the tests still continue to work.

Before merging, I will take care of removing them from the branch.

Reply:
I have re-added the Hatom tests. Seems that only 2 issues rename which I am 99% sure are from the Hatom Liquid staking algorithm.
You can search for `TODO` in the tests folder to find the two spots where calls were commented.

Other than that it seems the Chain Simulator is somehow really reliable now 😄 
Gas fees are consistent, didn't have issues with transactions remaining pending and then all executed after the required number of blocks, so I am pretty happy with the current outcome that everything seems to work.

Reply:
That's awesome news that it is now reliable! :fire: 

Will investigate more Hatom's tests